### DISCUSSION TITLE
Site as PWA

### DISCUSSION THREAD

**Discussion by anna-dingler** [hexojs/hexo#4954] [Question / Answer] [answered]
Hello!

I was wondering if Hexo has the ability to create PWAs or if there are any plans to add this feature. Thanks!

**Comment by lorezyra** [comment]
> I was wondering if Hexo has the ability to create PWAs or if there are any plans to add this feature. Thanks!

PWA? "Progressive Web App?"
I'm sure you could put the generated pages in a container (Electron) for that...

**Reply by anna-dingler** [reply]
Hi, I've already finished updating our site. Thank you!

**Comment by lorezyra** [comment, marked as answer]
Also look at: https://github.com/lavas-project/hexo-pwa and https://kiko.io/post/A-New-Blog-Customizing-Hexo/

**Comment by uiolee** [comment]
It can be done by plugins.
